tiledb/CMakeLists.txt

```diff
@@ -1169,3 +1178,14 @@ configure_file(
 )
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/tiledb.pc
         DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
+
+if (WIN32)
```

Comment:
Suggest to move this up to where it is used.

Reply:
moved (required moving the version values code up as well)